### ISSUE TITLE
`jsipfs add` fails on inputs larger than 1GiB when `rabin` is specified

### ISSUE THREAD

**Issue by ribasushi** [ipfs/js-ipfs-unixfs#41]
Reproducible as:
```
~$ for onemegs in 1022 1023 1024 1025 ; do echo -n "$(( $onemegs * 1048576 )) bytes: "; dd if=/dev/zero bs=1048576 count=$onemegs 2>/dev/null | jsipfs add -n -q --chunker=rabin ; done
1071644672 bytes: QmQtWcWN3DKaRnPDYU7nsPEbrSH63Py7LFQfFJ7bKaWHNi
1072693248 bytes: QmNex7npXYN4CFR4ysL6VsBLvAt9CQjAiSpvTnGMiQhZ9k
1073741824 bytes: unreachable
1074790400 bytes: unreachable

```

```
~$ jsipfs --version
0.40.0
```

**Comment by ribasushi** [comment]
> rabin outright does not converge with goipfs for a number of inputs

That turned out to be https://github.com/ipfs/js-ipfs-unixfs/issues/50

The original bug ( crash on > ~1GiB files ) seems to have been fixed by @hugomrdias' changes 🎉 
I will close this ticket once the full convergence test finishes ( ~3 hours )

**Comment by ribasushi** [comment]
This particular issue seems solved for good on `0.42.2-rc.4`. Finding others sadly, but will make separate tickets for them shortly.
